glove/glove.py

```diff
@@ -93,11 +96,12 @@ def fit(self, matrix, epochs=5, no_threads=2, verbose=False):
             raise Exception('Coocurrence matrix must be in the COO format')
 
         random_state = check_random_state(self.random_state)
-        self.word_vectors = ((random_state.rand(shape[0],
-                                                self.no_components) - 0.5)
-                             / self.no_components)
-        self.word_biases = np.zeros(shape[0],
-                                    dtype=np.float64)
+        if initial_epoch == 0:
+            self.word_vectors = ((random_state.rand(shape[0],
+                                                    self.no_components) - 0.5)
+                                 / self.no_components)
+            self.word_biases = np.zeros(shape[0],
+                                        dtype=np.float64)
 
         self.vectors_sum_gradients = np.ones_like(self.word_vectors)
         self.biases_sum_gradients = np.ones_like(self.word_biases)
```

Comment:
Please vectors_sum_gradients and biases_sum_gradients put in the if statement
``` python
if initial_epoch == 0:
        ...
        self.vectors_sum_gradients = np.ones_like(self.word_vectors)
        self.biases_sum_gradients = np.ones_like(self.word_biases)
```
Because using it to update learning rate in fit_vectors